### ISSUE TITLE
ModuleNotFoundError: No module named 'django.core.urlresolvers'

### ISSUE THREAD

**Issue by jedie** [edoburu/django-oscar-docdata#5]
with django v2.0:
```
  File "/usr/local/lib/python3.6/site-packages/oscar_docdata/dashboard/app.py", line 3, in <module>
    from . import views
  File "/usr/local/lib/python3.6/site-packages/oscar_docdata/dashboard/views.py", line 2, in <module>
    from django.core.urlresolvers import reverse_lazy, reverse
ModuleNotFoundError: No module named 'django.core.urlresolvers'
```

**Comment by jedie** [comment]
```
  File "/usr/local/lib/python3.6/site-packages/oscar_docdata/urls.py", line 2, in <module>
    from .views import OrderReturnView, StatusChangedNotificationView
  File "/usr/local/lib/python3.6/site-packages/oscar_docdata/views.py", line 6, in <module>
    from oscar_docdata import appsettings
  File "/usr/local/lib/python3.6/site-packages/oscar_docdata/appsettings.py", line 7, in <module>
    from django.core.urlresolvers import reverse_lazy
ModuleNotFoundError: No module named 'django.core.urlresolvers'
```

**Comment by vdboor** [comment]
I've fixed the basic compatibility with Django 2.0 in the git master branch. If you still find anything, please offer a pull request! Best, Diederik
